module/cyphersystem.js

```diff
@@ -963,28 +963,28 @@ function itemRollMacro (actor, itemID, pool, skill, assets, effort1, effort2, ad
   let pointsPaid = true;
 
   if (!actor || actor.data.type != "PC") {
-    return ui.notifications.warn(`This macro can only be used by ${owner.name}.`)
+    return ui.notifications.warn(`${game.i18n.localize("CYPHERSYSTEM.MacroOnlyUsedBy")} ${owner.name}.`)
```

Comment:
Needs a better solution, again ideally with nested translations. German needs a different sentence structure here.

Reply:
Will try to investigate further, I may have missed a way to do so

Reply:
Refactored translation